dds_web/user.py

```diff
@@ -59,6 +61,68 @@ def login():
         return redirect(to_go_url)
 
 
+def do_login(session, identifier:str, password:str = "") -> bool:
+    """
+    Check if a user with matching identifier exists. If so, log in as that user.
+
+    TODO:
+      * Add support for passwords
```

Comment:
How do you mean?

Reply:
I intend that function to be the way to log in, but I didn't spend the time to make it compatible with the existing password login